vNext/extensions/applicationinsights-analytics-js/src/JavaScriptSDK/ApplicationInsights.ts

```diff
@@ -565,6 +567,22 @@ export class ApplicationInsights implements IAppInsights, ITelemetryPlugin, IApp
         this._isInitialized = true;
     }
 
+    /**
+     * Log a page visit time
+     * @param    pageName    Name of page
+     * @param    pageVisitDuration Duration of visit to the page in milleseconds
+     */
+    private trackPageVisitTime(pageName: string, pageUrl: string, pageVisitTime: number) {
+        var properties = { PageName: pageName, PageUrl: pageUrl };
+        this.trackMetric({
```

Comment:
What is this metric?, is this something available before?

Reply:
Yes, it was in previous SDK version
https://github.com/Microsoft/ApplicationInsights-JS/blob/3ea7cd9839277802de0b99c8250062f3b752c1c7/JavaScript/JavaScriptSDK/AppInsights.ts#L425-L434